modules/custom-status/custom-status.php

```diff
@@ -405,7 +405,7 @@ function post_admin_header() {
 			// Load the custom statuses
 			foreach( $custom_statuses as $status ) {
 				$all_statuses[] = array(
-					'name' => esc_js( $status->name ),
+					'name' => esc_js( esc_html__( $status->name, 'edit-flow') ),
```

Comment:
Same as above regarding the translation being added properly but I question the esc_html__() and the esc_js(). I see it's currently present but I'm not sure I understand the purpose of the esc_js(). Here's a bit more information on https://vip.wordpress.com/2015/03/25/preventing-xss-in-javascript/